helm-charts/submarine/install.sh

```diff
@@ -0,0 +1,20 @@
+#
```

Comment:
Do we really need this file? can we remove it

Reply:
No, we don't really need it. Just for install convenience.